### DISCUSSION TITLE
Support for EC Encryption Algorithms for SSH Keys in Okteto

### DISCUSSION THREAD

**Discussion by swageblock** [okteto/okteto#4368] [Ideas]
As of Visual Studio 2022 version 17.10, [only EC (Elliptic Curve) keys are supported for remote debugging sessions](https://learn.microsoft.com/en-us/cpp/linux/connect-to-your-remote-linux-computer?view=msvc-170). This change impacts many developers who rely on RSA keys for their development workflows in Visual Studio, including those using Okteto.

**Benefits:** 

- Compatibility with a wide set of current developer tools
- Stronger security

**Special Considerations**

- It may be necessary to have a configurable solution for environment compatibility.

While the driver for this idea is the change that Microsoft has made to their IDE, this will be a valuable feature for maintaining the efficiency and security of development workflows for all Okteto users.  Thanks for considering this feature.

**Comment by swageblock** [comment]
It appears this may be resolved in Okteto CLI 2.31.0.

**Reply by codyjlandstrom** [reply]
Thanks for closing @swageblock. Let us know if you have any questions with the updated implementation!
